README.md

```diff
@@ -1,46 +1,103 @@
-# SecretHub Cross Language Golang Client<sup>**experimental**</sup>
+# SecretHub-XGO Guide for C# integration
 
-`secrethub-xgo` wraps the `secrethub-go` client with `cgo` exported functions so it can be called from other languages, e.g. C, C#, Python, Ruby, NodeJS, and Java. 
+This is a guide containing the following:
+ - [Prerequisites](#prerequisites)
+ - [Usage](#usage)
+ - [Building from source](#building-from-source)
+ - [How to call library functions](#how-to-call-library-functions)
+ - [Resources](#resources)
+ - [Getting help](#getting-help)
 
```

Comment:
I think this is still as if it's a guide, not the readme.

Reply:
Adjusted. Let me know if it is ok.